tests/Admin/AdherentAdminTest.php

```diff
@@ -35,6 +36,78 @@ public function testAnAdminCantBanAnAdherent()
         $this->assertResponseStatusCode(404, $this->client->getResponse());
     }
 
+    public function testEditBoardMemberInformations()
+    {
+        $adherent = $this->adherentRepository->findOneByUuid(LoadAdherentData::ADHERENT_2_UUID);
+        $this->assertTrue($adherent->isBoardMember());
+        $this->authenticateAsAdmin($this->client);
+        $editUrl = sprintf('/admin/app/adherent/%s/edit', $adherent->getId());
```

Comment:
The URI is used at least twice, i would place her in a class constante. WDYT?

Reply:
why not, done